.github/workflows/aws-deploy.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         shell: bash
-        if: ${{ env.WORKFLOW_USE_ARTIFACTS == '' }}
+        if: ${{ inputs.WORKFLOW_USE_ENV == '' }}
         run: |
```

Comment:
Unrelated to the change, but this script seems exactly the same as the previous one ("with .build_env") – is this expected?